### PR TITLE
Clarify Proxmox LXC instructions; reference #5773 instead of #1111

### PR DESCRIPTION
**Pull request by m-d-brown**
There is a lot of confusing information online. This provides some simple general guidelines to help people get started.

This is a recreation of #10747.

### CODE REVIEW

File: docs/docs/frigate/installation.md [hunk 1]
```diff
@@ -202,7 +202,16 @@ To install make sure you have the [community app plugin here](https://forums.unr
...
+
+- For Intel-based hardware acceleration, to allow access to the `/dev/dri/renderD128` device with major number 226 and minor number 128, add the following lines to the `/etc/pve/lxc/<id>.conf` LXC configuration:
+  - `lxc.cgroup2.devices.allow: c 226:128 rwm`
+  - `lxc.mount.entry: /dev/dri/renderD128 dev/dri/renderD128 none bind,optional,create=file`
+- The LXC configuration will likely also need `features: fuse=1,nesting=1`. This allows running a Docker container in an LXC container (`nesting`) and prevents duplicated files and wasted storage (`fuse`).
+- Successfully passing hardware devices through multiple levels of containerization (LXC then Docker) can be difficult. Many people make devices like `/dev/dri/renderD128` world-readable in the host or run Frigate in a privileged LXC container.
+- The virtualization layer often introduces a sizable amount of overhead for communication with Coral devices, but [not in all circumstances](https://github.com/blakeblackshear/frigate/discussions/1837).
+
+See the [Proxmox LXC discussion](https://github.com/blakeblackshear/frigate/discussions/1111) for more general information.
```
Comment (NickM-27) [inline docs/docs/frigate/installation.md:214]: Looks like this is still pointing to the old outdated guide

File: docs/docs/frigate/installation.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Frigate is a Docker container that can be run on any Docker host including as a
 
 ### Operating System
 
-Frigate runs best with docker installed on bare metal debian-based distributions. For ideal performance, Frigate needs access to underlying hardware for the Coral and GPU devices. Running Frigate in a VM on top of Proxmox, ESXi, Virtualbox, etc. is not recommended. The virtualization layer often introduces a sizable amount of overhead for communication with Coral devices, but [not in all circumstances](https://github.com/blakeblackshear/frigate/discussions/1837).
+Frigate runs best with Docker installed on bare metal Debian-based distributions. For ideal performance, Frigate needs low overhead access to underlying hardware for the Coral and GPU devices. Running Frigate in a VM on top of Proxmox, ESXi, Virtualbox, etc. is not recommended though [many people have had success with Proxmox](#proxmox).
```
Comment (NickM-27) [inline docs/docs/frigate/installation.md:16]:
```suggestion
Frigate runs best with Docker installed on bare metal Debian-based distributions. For ideal performance, Frigate needs low overhead access to underlying hardware for the Coral and GPU devices. Running Frigate in a VM on top of Proxmox, ESXi, Virtualbox, etc. is not recommended though [some users have had success with Proxmox](#proxmox).
```

**Review by NickM-27 — CHANGES_REQUESTED** [review]
Couple small things. Looks like checks are failing too

File: docs/docs/frigate/installation.md (hunk 1 above)
Comment (m-d-brown) [reply docs/docs/frigate/installation.md]: There are several guides and #1111 is the latest I can find in the docs. I'm now referencing #5773 here. If you have another mind, please let me know.

**Comment by m-d-brown** [comment]
> Couple small things. Looks like checks are failing too

docker/main/rootfs/usr/local/go2rtc/create_config.py is failing ruff formatting checks prior to this PR. Do you want me to fix it here or fix it separately?

**Comment by NickM-27** [comment]
Ahh I forgot it's that, yeah that's expected so no need to worry about the checks.

**Review by NickM-27 — APPROVED** [review]

**Comment by m-d-brown** [comment]
Hi @NickM-27, do you merge as the next step? Or perhaps I'm missing something.

**Comment by NickM-27** [comment]
Another maintainer will review and merge

**Pull request merged**